### PR TITLE
Hero Responsiveness Added

### PR DESCRIPTION
**Pull request by Milcke**

### CODE REVIEW

File: app/(landing)/components/SecondComp.tsx [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
-                <Image
-                    src="/main.svg"
-                    alt='Image'
-                    width={0}
-                    height={0}
-                    sizes="100vw"
-                    className='drop-shadow-lg'
-                    style={{ width: '100%', height: 'auto' }} // optional
-                />
+const SecondComponent = () => {
```
Comment (aftabrehan) [inline app/(landing)/components/SecondComp.tsx]: SecondComponent? Why don't we give it a name?

File: app/(landing)/components/TopComponent.tsx [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
+    const session = await getServerAuthSession();
+    return (
+        <div className="mt-[8.5rem] flex justify-center items-center bg-cover bg-no-repeat">
+            <div className="container mx-auto px-4 sm:px-6 lg:px-8">
+                <div className="text-center flex flex-col w-full h-full">
+                    <span className="text-white text-6xl sm:text-6xl lg:text-[95px] font-bold font-manrope leading-tight">One Link with</span>
+                    <span className="bg-gradient-to-r from-[#FF5400] via-[#FF5400] to-[#FF0054] text-transparent bg-clip-text text-6xl sm:text-6xl lg:text-8xl font-bold font-manrope ">Superpowers</span>
+                </div>
+                <div className='flex flex-col space-y-3 justify-center items-center'>
+                    <div className="text-center max-w-3xl text-stone-400 text-base px-4 sm:text-lg lg:text-xl mt-4 sm:mt-6 lg:mt-12 font-medium font-manrope leading-snug">xProfile is an open-source app that provides a one-link portfolio for showcasing your projects, skills, social links, and more.</div>
```
Comment (aftabrehan) [inline app/(landing)/components/TopComponent.tsx]:
```suggestion
                    <div className="text-center max-w-3xl text-stone-400 text-base px-4 sm:text-lg lg:text-xl mt-4 sm:mt-6 lg:mt-12 font-medium font-manrope leading-snug">XProfile is an open-source app that provides a one-link portfolio for showcasing your projects, skills, social links, and more.</div>
```

File: app/(landing)/components/CTAbtn.tsx [hunk 3]
```diff
@@ -66,8 +65,8 @@ const CTAbtn = ({
                     </Button>
                 ) : (
                     <Form {...form}>
-                        <form onSubmit={form.handleSubmit(onSubmit)} className='flex gap-2 mt-[68px] justify-center items-center'>
-                            <div className="flex rounded-xl">
+                        <form onSubmit={form.handleSubmit(onSubmit)} className='flex flex-col gap-0 md:gap-4 mt-4 md:mt-12 md:flex-row md:items-center '>
```
Comment (aftabrehan) [inline app/(landing)/components/CTAbtn.tsx]: I guess by default the gap is 0, so we don't need to add a class explicitly, right?

```suggestion
                        <form onSubmit={form.handleSubmit(onSubmit)} className='flex flex-col md:gap-4 mt-4 md:mt-12 md:flex-row md:items-center'>
```

File: app/(landing)/components/CTAbtn.tsx [hunk 4]
```diff
@@ -45,12 +44,12 @@ const CTAbtn = ({
     }
 
     // Completed Form ?
-    const isProfile = session?.user?.completedProfile
+    const isProfile = session?.user?.completedProfile;
```
Comment (aftabrehan) [inline app/(landing)/components/CTAbtn.tsx:47]: Can you please add a line between `return` and the variables, just for the readablity.

```suggestion
    const isProfile = session?.user?.completedProfile;
    
```

File: app/(landing)/components/SecondComp.tsx [hunk 5]
```diff
@@ -1,29 +1,31 @@
...
+    return (
+        <div className='mt-8 md:mt-24  overflow-hidden'>
+            <div className="max-w-screen-md mx-5 rounded-3xl border p-2 md:p-4 flex-shrink-0 bg-neutral-500/10 text-stone-400 bg-opacity-30 backdrop-blur-3xl border-t border-b border-l">
+                <div className="flex md:flex-row gap-2 md:gap-4 mb-2 px-1">
+                    <svg className="w-7 h-8 md:w-10 md:h-12" viewBox="0 0 55 53" fill="none" strokeWidth="2.75" xmlns="http://www.w3.org/2000/svg">
+                        <rect x="1.35587" y="2.08487" width="52.0653" height="48.8112" stroke="currentColor" strokeWidth="2.71173" />
+                        <path stroke="currentColor" fill="currentColor" strokeWidth="1" d="M34.7813 39.1532H28.464L22.8063 30.0938L17.2527 39.1532H11.8032L20.151 26.4146L12.0635 14.0577H18.3287L23.3443 22.1799L28.3078 14.0577H33.6879L25.9476 25.7551L34.7813 39.1532ZM37.4469 39.1532V33.6343H43.0006V39.1532H37.4469Z" />
+                    </svg>
```
Comment (aftabrehan) [inline app/(landing)/components/SecondComp.tsx]: Using svg like this isn't a good practice. Can we create an svg file and import it as a React component and then use it?

**Review by aftabrehan — CHANGES_REQUESTED** [review]
Excellent PR, @Milcke 💪 🚀 💯 

It just needs minor adjustments and then we're good to go.

**Comment by Milcke** [comment]
@aftabrehan Fixed the issues - thanks!

File: app/(landing)/components/SecondComp.tsx (hunk 1 above)
Comment (Milcke) [reply app/(landing)/components/SecondComp.tsx]: changed in commit

File: app/(landing)/components/SecondComp.tsx (hunk 5 above)
Comment (Milcke) [reply app/(landing)/components/SecondComp.tsx]: changed in commit

File: app/(landing)/components/HeroSection.tsx [hunk 6]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import LandingNav from '@/components/LandingNav';
 import Round from './round';
 import TopComponent from './TopComponent';
-import SecondComponent from './SecondComp';
+import SecondComponent from './ProfileExample';
```
Comment (aftabrehan) [inline app/(landing)/components/HeroSection.tsx:5]:
```suggestion
import ProfileExample from './ProfileExample';
```

File: components/icons.tsx [hunk 7]
```diff
@@ -219,5 +219,12 @@ export const Icons = {
       </defs>
     </svg>
   ),
+  boxlogo: (props: IconProps) => (
```
Comment (aftabrehan) [inline components/icons.tsx:222]: That's a much better approach.

**Review by aftabrehan — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**